src/base/logging.hh

```diff
@@ -123,19 +126,41 @@ class Logger
      */
     [[noreturn]] void exit_helper() { exit(); ::abort(); }
 
+    void
+    registerExtraLog(std::function<std::string()> cb)
```

Comment:
This function was added to the public API, and as such respective unit tests should be added (src/base/logging.test.cc)

Reply:
Test added in logging.test.cc. Please check :)